compiler/test/dotty/tools/repl/ReplCompilerTests.scala

```diff
@@ -115,4 +115,9 @@ class ReplCompilerTests extends ReplTest {
     //   storedOutput().startsWith("scala.MatchError: null")
     // }
   }
+
+  @Test def i2789: Unit = fromInitialState { implicit state =>
+    compile("(x: Int) => println(x)")
```

Comment:
Why not put this in a repl script test?

Reply:
Because it prints something like this:
```scala
val res0: Int => Unit = Lambda$1423/1112073835@3e6534e7
```
which is not deterministic